podcasts/New Search/Views/Results/SearchResultCell.swift

```diff
@@ -33,13 +33,16 @@ struct SearchResultCell: View {
             VStack(alignment: .leading, spacing: 12) {
                 HStack(spacing: 12) {
                     if result?.kind == .podcast || episode != nil {
-                        PodcastCover(podcastUuid: episode?.podcastUuid ?? result?.uuid ?? "")
-                            .frame(width: 48, height: 48)
+                        PodcastImage(uuid: episode?.podcastUuid ?? result?.uuid ?? "")
+                            .frame(width: 56, height: 56)
+                            .cornerRadius(4)
+                            .shadow(radius: 3, x: 0, y: 1)
                             .allowsHitTesting(false)
                     } else if let result {
                         SearchFolderPreviewWrapper(uuid: result.uuid)
-                            .frame(width: 48, height: 48)
+                            .frame(width: 56, height: 56)
                             .cornerRadius(4)
+                            .shadow(radius: 3, x: 0, y: 1)
                             .allowsHitTesting(false)
```

Comment:
If we decide to go with the common view we can replace the code here with something like: 

```
                    (episode?.podcastUuid ?? result?.uuid).map {
                        SearchEntryImage(uuid: $0, kind: result?.kind)
                    }
```

Reply:
Done in f6d977fcdc03ed9200c138469a2fec491959e3fd